src/service/resource.js

```diff
@@ -556,9 +557,10 @@ export class Resource {
    * @return {boolean}
    */
   isDisplayed() {
-    return (this.layoutBox_.height > 0 && this.layoutBox_.width > 0 &&
+    return (this.element.getLayout() == Layout.FLUID ||
```

Comment:
Please break this into multiple checks.

Reply:
Using variables for clarity. If this isn't what you meant, please be more specific.